### PR TITLE
Feature/flask setups

### PR DESCRIPTION
**Pull request by Zachariasp**
I added some additional steps to help us make sure our machines are setup the same and to instruct on how to start the app.

### CODE REVIEW

File: api/setup.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        'idna~=3.3',
+        'itsdangerous~=2.1.2',
+        'Jinja2~=3.1.2',
+        'MarkupSafe~=2.1.1',
+        'requests~=2.28.1',
+        'urllib3~=1.26.11',
+        'Werkzeug~=2.2.0',
+        'Wikipedia-API~=0.5.4'
+    ],
+    # extras_require={
```
Comment (Owlgaze) [inline api/setup.py:27]: please remove commented code.

**Review by Owlgaze — CHANGES_REQUESTED** [review]
LGTM! minor request to remove commented code, but other than that, we're lookin peachy~

File: api/setup.py (hunk 1 above)
Comment (Zachariasp) [reply api/setup.py]: deleted

**Review by Owlgaze — APPROVED** [review]
h e l l  y e a h

**Pull request merged**